.github/workflows/publish.yaml

```diff
@@ -0,0 +1,58 @@
+name: Tag and Publish
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Pull latest changes
+        run: git pull origin main
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to Github Packages
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build image and push to GitHub Container Registry
+        uses: docker/build-push-action@v3
+        with:
+          # relative path to the place where source code with Dockerfile is located
+          context: .
+          push: true
+          tags: ghcr.io/allenneuraldynamics/anivia:latest
+  update_ecs_service :
+    runs-on: ubuntu-latest
+    needs: publish
+    permissions:
+      id-token: write
+      contents: read
+    env:
+      ECS_ROLE: ${{ secrets.AWS_ANIVIA_ECS_ROLE }}
```

Comment:
Do these need to be added to GitHub?

Reply:
Yes, and I will set those variables prior to merging. I've already created the role in AWS dev.